frontend/src/components/ExperimentCollection/ExperimentCollectionDashboard/ExperimentCollectionDashboard.scss

```diff
@@ -0,0 +1,7 @@
+.aha__dashboard {
+
+    .aha__logo {
+        margin-left: 35%;
```

Comment:
What's the reason for using a percentage as margin? Should it scale with device width?